data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+			"Do you see that Hai woman carrying her baby? Doesn't it look like a long-tailed teddy-bear? The Hai are so cute."
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "concerned merchant in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Merchant captain"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I took a shortcut through Wah Ki with six ships. Just two seconds later, BAM! half my fleet is dust."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
+			"These Hai are so creepy. What are they hiding? I can't wait to get back to human space."
+			"The Hai don't like my ship's armory of laser rifles, but I was once boarded by a pirate crew and I'm never letting that happen again! If some overgrown squirrel is going to take away my rifle, it'll have to be from my cold dead hands!"
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
+			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on this stuff."
+			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "human citizen in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Human citizen"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
+			"My Hai friend went on one of those prison cruises a while back, but it got lost in an ion storm near Allhome."
+			"One of my Hai co-workers went to Sunracer to formalize a business deal. I warned her that part of human space is dangerous. I hope I'll see her again."
+			"A Hai that lives near me went to sleep one night and woke up in a box on Prime. Isn't that weird?"
```

Comment:
What?

Reply:
Deleted.

The kid from Allhome that ended up in a box wasn't the only one. It is to set up lore for the Philosopher's Paradise. I see how this one looks bizarre and out-of-place though.